dart/example/event_example.dart

```diff
@@ -5,15 +5,15 @@ final event = SentryEvent(
   serverName: 'server.dart',
   release: '1.4.0-preview.1',
   environment: 'Test',
-  message: Message('This is an example Dart event.'),
+  message: SentryMessage('This is an example Dart event.'),
   tags: const <String, String>{'project-id': '7371'},
   extra: const <String, String>{'section': '1'},
-  fingerprint: const <String>['example-dart'],
-  user: User(
+  // fingerprint: const <String>['example-dart'],
+  user: SentryUser(
     id: '800',
     username: 'first-user',
     email: 'first@user.lan',
-    ipAddress: '127.0.0.1',
+    // ipAddress: '127.0.0.1',
```

Comment:
I enabled sendDefaultPii

Reply:
I think it would make sense to write this as a comment in the example.